alfresco-googledrive-share/src/main/java/org/alfresco/integrations/web/evaluator/SizeLimitEvaluator.java

```diff
@@ -107,23 +101,17 @@ private String getContentType(String mimetype)
         return null;
     }
 
-
     private long getMaxFileSize(String contentType)
     {
-        if (contentType.equals("document"))
+        switch (contentType)
         {
+        case "document":
```

Comment:
Should we use existing constants, eg.
https://github.com/Alfresco/googledrive/blob/master/alfresco-googledrive-repo-base/src/main/java/org/alfresco/integrations/google/docs/GoogleDocsConstants.java#L45-L48

As an aside: also came up in a recent separate PR comment (fixed in a different way), eg. https://github.com/Alfresco/googledrive/pull/75#discussion_r356742046

Reply:
I've declared separate constants in this class as the ones you referenced are in a different module (_...repo-base_), which is not a dependency of the current one (_...share_).